### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by marians**
As always, the PR template should ensure that people follow required steps for the repo.

In this case, it informs about the specific versioning guidelines and schema-specific READMEs.

### CODE REVIEW

**Review by kuosandys — APPROVED** [review]

**Comment by marians** [comment]
@vvondruska @gusevda In this repo I need a second review (because I configured it to be super safe...). Can you help me please?

**Review by mogottsch — APPROVED** [review]

**Pull request merged**
